tests/test_other.py

```diff
@@ -63,6 +63,16 @@ def basic_good_with_trailing_empty_cols(tmpdir):
     yield p.as_posix()
 
 
+@pytest.fixture()
+def basic_good_with_trailing_empty_cols_and_rows(tmpdir):
```

Comment:
codecov is complaining because this test fixture doesn't actually get invoked. Add that in to your tests and make sure that you're actually getting the desired behavior.

Reply:
Ah, whoops, I had been reusing an old test then decided last moment to add it as a separate test, and didn't follow it through. Should be fixed now